pkg/cli/verifier/envoy_config.go

```diff
@@ -429,12 +432,73 @@ func (v *EnvoyConfigVerifier) verifyDestination() Result {
 			result.Reason = fmt.Sprintf("Did not find matching inbound cluster for service %q: %s", dst, err)
 			return result
 		}
+
 		// Verify server TLS secret
 		if err := v.findTLSSecretsOnDestination(secrets); err != nil {
 			result.Status = Failure
 			result.Reason = fmt.Sprintf("Server TLS secret not found for pod %q: %s", v.configAttr.trafficAttr.DstPod, err)
 			return result
 		}
+
+		// Next, if ingress is enabled, check for ingress filters
+		// TODO: Add check for ingress filters
+		if v.configAttr.trafficAttr.IsIngress {
+			src := v.configAttr.trafficAttr.SrcService
+			// grab endpoints for ingress service
+			endpoints, err := v.kubeClient.CoreV1().Endpoints(src.Namespace).Get(context.Background(), src.Name, metav1.GetOptions{})
+			if err != nil {
+				result.Status = Failure
+				result.Reason = fmt.Sprintf("Ingress source service %q not found: %s", src, err)
+				return result
+			}
+
+			sourceIPs := map[string]bool{}
+			filterChainName := fmt.Sprintf("ingress_%s/%s_%d_%s", svc.GetName(), svc.GetNamespace(), v.configAttr.trafficAttr.DstPort, v.configAttr.trafficAttr.AppProtocol)
```

Comment:
Would be nice to use a helper for this. I'd add a helper in `pkg/trafficpolicy/ingress_types.go` called `GetIngressTrafficMatchName(svcName, port, protocol)` and this code and the code in `pkg/catalog/ingress.go` can reuse to set the filter_chain_match.name